### DISCUSSION TITLE
Uso da linguagem neutra nas comunicações oficiais na APyB

### DISCUSSION THREAD

**Discussion by tiidadavena** [apyb/comunidade#64] [Geral]
Tópico iniciado como desdobramento da assembleia geral (#43 ).

Apesar da atual diretoria da APyB estar preconizando pelo uso da linguagem neutra, entendemos que a comunjdade podem ajudar a impulsionar e a levantar mais pontos de melhoria.

Outra motivação seria avaliar a demanda a comunidade acerca do tema e da padronização dos canais de comunicação.

**Comment by luzfcb** [comment]
Embora eu já tenho ouvido falar do termo, acho que nem todo mundo pode saber a _Definição Formal_, como se usa, quando se usa e quando não se usa linguagem neutra e etc.
Penso que seria bom anexar a esse tópico algo com a _Definição Formal_ de linguagem neutra, assim dando a oportunidade para os associados que não tem conhecimento profundo sobre o tema tenham a oportunidade de aprender.

**Reply by gabubellon** [reply]
Não sei se existe uma definição formal de linguagem neutra, uma vez que a lingua brasileira tem sua neutralidade na maioria das vezes no feminino.
Acho que o ideal seria criar um estilo de escrita não formal da comunidade.
Uma coisa que podemos fazer é nos espelhar em outras comunidades.

**Comment by gabubellon** [comment]
Estava tentando achar um artigo antigo que eu tinha aqui e achei esse https://medium.com/guia-para-linguagem-neutra-pt-br/guia-para-linguagem-neutra-pt-br-f6d88311f92b que parece mais completo
